server/src/main/java/io/druid/server/security/ResourceType.java

```diff
@@ -0,0 +1,6 @@
+package io.druid.server.security;
+
+public enum ResourceType
```

Comment:
@drcrallen i think, the set of valid resource types would be fixed by druid-core because druid-core calls in to the action and provides those as arguments, so it makes sense to use enum to represent resource types.
in any case, since we are calling the api experimental and potentially changeable in near future. I wouldn't worry too much about it either ways.